### ISSUE TITLE
[JENKINS-264] Mark projects a exclusive

### ISSUE THREAD

**Issue by timja-bot** [timja-org/jenkins-gh-issues-poc-11-04#9601]
<p>I'd like to mark a project as "exclusive". No other project that is marked as<br/>
exclusive can be run at the same time.</p>

<p>This is useful for projects that require exclusive access to resources i.e.<br/>
network ports</p>

<p>Resolving this issue would allow me to build my target containers in parallel.<br/>
Tests however need to run sequetially.</p>

<p>cheers<br/>
-thomas</p>

---
<details><summary><i>Originally reported by <a title="Thomas Diesler" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=tdiesler">tdiesler</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-264" target="_blank">Mark projects a exclusive</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Major
<li><b>resolution</b>: Cannot Reproduce
<li><b>resolved</b>: 2009-03-07T09:07:52+00:00
<li><b>imported</b>: 2025-11-05
</ul></i>
</details>

**Comment by timja-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kohsuke">kohsuke</a>:</i>
<p>I suspect this is a duplicate of issue #220. Is this not?</p>
